frontend/plugins/nuxt/appInit.ts

```diff
@@ -0,0 +1,38 @@
+import isNil from 'lodash/isNil';
+import { authStore, ServerInfo } from '~/store';
+import { parseServerListString } from '~/utils/servers';
+
+export default function (): void {
+  const auth = authStore();
+
+  /**
+   * Initializes Axios and the websocket with the current data in the store
+   */
+  auth.authInit();
+
+  /**
+   * Logout the user on app initialization if the user doesn't want to be remembered.
+   *
+   * The logic to handle logouts and user switches during runtime lives inside Pinia watchers (~/plugins/store/watchers/auth)
+   * Middleware lives at ~/middleware/auth
+   */
+  if (!auth.rememberMe) {
+    auth.logoutUser();
+  }
+
+  /**
+   * Add default servers from environment variables
+   */
+  const defaultServers = parseServerListString(process.env.server_url_list);
+  const missingServers = defaultServers.filter((serverUrl) => {
+    const server = auth.servers.find(
+      (lsServer: ServerInfo) => lsServer.PublicAddress === serverUrl
+    );
+
+    return !isNil(server);
```

Comment:
```suggestion
    return isNil(server);
```
### Related to #1698

If you are searching the Missing servers, the check should be on isNil intead of it's opposite.

If the result of the find() is undefined then the server is missing.

Ps. i have tryied out in my local env before telling you this

Reply:
Ooops, good catch!